samples/sensor/veml7700/sample.yaml

```diff
@@ -0,0 +1,9 @@
+sample:
+  name: VEML7700 Sensor sample
+tests:
+  sample.sensor.bme680:
+    harness: sensor
+    tags: samples sensor
+    integration_platforms:
+      - nrf52840dk_nrf52840
+    platform_allow: nrf52840dk_nrf52840
```

Comment:
sample is not specific to nordic boards, so platform_allow should not be set, please filter by dt_compat_enabled (see some other samples). I assume the sensor needs to be manually wired to this board (as it's not part of the board), so I'm not sure making this board an integration platform is a good idea.